parlai/core/opt.py

```diff
@@ -131,3 +132,15 @@ def load(cls, optfile: str) -> 'Opt':
             if key in dct:
                 del dct[key]
         return cls(dct)
+
+    def log(self, header="Opt"):
+        from parlai.core.params import print_git_commit
```

Comment:
any reason to keep `print_git_commit` in `parlai.core.params`?

Reply:
Laziness. Where's the right place you think?

Maybe it could print parlai.__version__, which could have the git commit. I don't know how to implement that though.

Reply:
it feels like a utils function - pytorch used to have a `version.py` file but I'm not sure where that went. I'm overall indifferent as to where it should live